cassandra/util.py

```diff
@@ -1011,3 +1011,86 @@ def __str__(self):
         except:
             # If we overflow datetime.[MIN|MAX]
             return str(self.days_from_epoch)
+
+import socket
+if hasattr(socket, 'inet_pton'):
+    inet_pton = socket.inet_pton
+    inet_ntop = socket.inet_ntop
+else:
+    """
+    Windows doesn't have socket.inet_pton and socket.inet_ntop until Python 3.4
+    This is an alternative impl using ctypes, based on this win_inet_pton project:
+    https://github.com/hickeroar/win_inet_pton
+    """
+    import ctypes
+
+
+    class sockaddr(ctypes.Structure):
+        _fields_ = [("sa_family", ctypes.c_short),
+                    ("__pad1", ctypes.c_ushort),
```

Comment:
Might want to comment here that "__pad1" skips over the "u_short sin_port" for v4 and "ipv4_addr" skips over the "u_long sin6_flowinfo" for v6 (from https://msdn.microsoft.com/en-us/library/windows/desktop/ms740496(v=vs.85).aspx). This is a bit clever. :)